bindings/rust/s2n-tls/Cargo.toml

```diff
@@ -10,6 +10,7 @@ license = "Apache-2.0"
 
 [features]
 default = []
+unstable-fingerprint = ["s2n-tls-sys/unstable-fingerprint"]
 quic = ["s2n-tls-sys/quic"]
```

Comment:
Do we need to include "unstable" in the name?

Reply:
I would vote yes. It relies on unstable features, and therefore is also unstable. https://github.com/aws/s2n-tls/pull/4009#discussion_r1204699991